content/zh/登记投票/dc.md

```diff
@@ -0,0 +1,15 @@
++++
+
+state_name = "District of Columbia"
+state_abbreviation = "dc"
+registration_type = "online"
+translationKey = "District of Columbia"
+hp_link = "https://dcboe.org/"
+registration_link = "https://forms.dc.gov/f/DC_Voter_Registration_Form"
```

Comment:
Update registration_link to `https://www.dcboe.org/Voters/Register-To-Vote/Register-to-Vote`